bootstrap/widgets.py

```diff
@@ -63,8 +63,10 @@ class BootstrapWidget (object):
 
     def build_attrs(self, *args, **kwargs):
         attrs = super(BootstrapWidget, self).build_attrs(*args, **kwargs)
-        if self.is_required:
+        if self.is_required and not isinstance(self, RadioSelect):
             attrs.update({'aria-required': 'true'})
+        if isinstance(self, RadioSelect):
```

Comment:
Should there be a `self.is_required` check here as well? Do radio inputs not support `aria-required`?